index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Capstone Web Page</title>
+</head>
+
+
+<body>
+    <header id="navbar">
+        <nav>
+            <div id="log-section">
+                <a href="#" id="my-page">My page</a>
+                <a href="#" id="log-in-out">Log in</a>
+            </div>
+            <div id="hamburger-btn" class="hhh" onclick="myFunction(this)">
+                <div class="bar1"></div>
+                <div class="bar2"></div>
+                <div class="bar3"></div>
+            </div>
+            <div id="menu-section">
+                <a href="index.html" class="logo"><img src="images/Logo.png" alt="M logo"></a>
+                <ul id="menu-items">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Course</a></li>
+                    <li><a href="#">Payment</a></li>
+                    <li><a href="#">FAQs</a></li>
+                </ul>
+                <a href="#" id="sign-in-btn">Sign in</a>
+            </div>
+        </nav>
+
+
+        <section id="mobile_menu">
+            <div><a href="index.html" class="logo"><img src="images/Logo.png" alt="M logo"></a></div>
+            <ul id="mobile_ul">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Course</a></li>
+                <li><a href="#">Payment</a></li>
+                <li><a href="#">FAQs</a></li>
+                <li><a href="#">Sign in</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log in</a></li>
+            </ul>
+        </section>
+    </header>
+
+
+    <section id="headline">
+        <p class="first-title">"Hello! Microverse Family"</p>
+        <h1 class="second-title">WELCOME TO MICROVERSE CAPSTONE PROJECT 2022</h1>
+        <p class="subtitle">In this capstone project, you are going to build is based on an online website for a conference. We provide some design guidelines for you to create the website, but you must personalize the content, i.e., instead of a conference you can build a website for a concert or for a web development course. It's very important the you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.</p>
+        <p class="dates">28.12.2022(WED) ~ 02.01.2023(MON)</p>
```

Comment:
- Make sure the top section includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a 👉  **smaller line of text**.
- 
![Capture](https://user-images.githubusercontent.com/63972670/210219682-1ba8e74a-c3e8-49f9-9e59-d463c90533eb.PNG)